### DISCUSSION TITLE
Trapped by a water current

### DISCUSSION THREAD

**Discussion by Johri** [FakeFishGames/Barotrauma#14820] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

very small local force/stream pushing my swimming crew members into a corner and get trapt there.

### Reproduction steps

1. Get a swimsuit 2. Get outside 3. Get in the rim 4. Find a spot with high stream drift 5. get pushed into a corner 6. die from suffocation

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Do you have the save file or a level seed available by any chance? I think it might be very difficult for us to find a spot where this occurs otherwise. :/ Thanks and sorry for for the inconvenience!
